### PR TITLE
Implement alert-group-settings APIs

### PR DESCRIPTION
**Pull request by xcezx**
This pull request implements alert-group-settings APIs.
refs. https://mackerel.io/api-docs/entry/alert-group-settings

### CODE REVIEW

File: alert_group_settings.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+		return nil, err
+	}
+	resp, err := c.Request(req)
+	defer closeResponse(resp)
+	if err != nil {
+		return nil, err
+	}
+
+	var data struct {
+		AlertGroupSettings []*AlertGroupSetting `json:"alertGroupSettings"`
```
Comment (xcezx) [inline alert_group_settings.go:34]: According to the API docs as follows:
> ```
> {
>   "alert_group_settings": [<alertGroupSetting>, <alertGroupSetting>, ...]
> }
> ```

but it's actually as follows:
```
{
  "alertGroupSettings": [<alertGroupSetting>, <alertGroupSetting>, ...]
}
```

Comment (astj) [reply alert_group_settings.go:34]: Right. We'll fix the docs soon.

Comment (astj) [reply alert_group_settings.go:34]: Deployed fix for [the API doc](https://mackerel.io/api-docs/entry/alert-group-settings#list).
ref: https://github.com/mackerelio/documents/commit/356018e3c99fe03b8e58218ccbe670dace8f526e

**Review by susisu — APPROVED** [review]
LGTM!
Thank you for your contribution, and sorry for the late review.

**Pull request merged**